### PR TITLE
simplify boilerplate with new helpers, ops

### PR DESCRIPTION
**Pull request by jhellerstein**
first cut. 

One concern is that recv_udp(0) doesn't have a way to tell us what port it ended up getting.

### CODE REVIEW

File: hydroflow_lang/src/graph/ops/recv_stdin.rs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+/// > 0 input streams, 1 output stream
+///
+/// > Arguments: port number
+///
+/// `recv_stdin` receives a Stream of lines from stdin
+/// and emits each of the elements it receives downstream.
+///
+/// ```rustbook
+/// let mut flow = hydroflow::hydroflow_syntax! {
+///     recv_stdin -> map(|x| x.to_uppercase())
```
Comment (MingweiSamuel) [inline hydroflow_lang/src/graph/ops/recv_stdin.rs:16]: `recv_stdin()`

File: hydroflow_lang/src/graph/ops/recv_udp.rs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    OperatorConstraints, OperatorWriteOutput, WriteContextArgs, WriteIteratorArgs, RANGE_0, RANGE_1,
+};
+
+use quote::quote_spanned;
+
+/// > 0 input streams, 1 output stream
+///
+/// > Arguments: port number
+///
+/// `recv_net` binds to a local network port and receives a Stream of serialized data from a remote sender,
```
Comment (MingweiSamuel) [inline hydroflow_lang/src/graph/ops/recv_udp.rs:11]: `recv_udp`?

File: hydroflow_lang/src/graph/ops/recv_udp.rs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+/// deserializes each element and emits each of the elements it receives downstream.
+///
+/// ```rustbook
+/// let mut flow = hydroflow::hydroflow_syntax! {
+///     recv_net(9000) -> map(|x| x.to_uppercase())
```
Comment (MingweiSamuel) [inline hydroflow_lang/src/graph/ops/recv_udp.rs:16]: `recv_udp(9000)`

**Review by MingweiSamuel — CHANGES_REQUESTED** [review]

File: hydroflow/examples/echoserver/client.rs [hunk 4]
```diff
@@ -1,54 +1,26 @@
-use crate::helpers::{deserialize_msg, resolve_ipv4_connection_addr, serialize_msg};
-use crate::protocol::{EchoMsg, EchoResponse};
-use crate::{GraphType, Opts};
-use hydroflow::hydroflow_syntax;
-use textnonce::TextNonce;
+use std::net::SocketAddr;
 use tokio::io::AsyncBufReadExt;
 use tokio::net::UdpSocket;
```
[the remark this answers is not in the document]
Comment (jhellerstein) [reply hydroflow/examples/echoserver/client.rs:2]: don't seem to be able to remove this despite the change above. let's try together

**Review by MingweiSamuel — APPROVED** [review]
LGTM. I think we will rename `sink_async_serde`/`recv_stream_serde` as part of #276 / #216

**Pull request merged**